cmd/oras/root/manifest/delete.go

```diff
@@ -86,6 +88,13 @@ func deleteManifest(ctx context.Context, opts deleteOptions) error {
 		return oerrors.NewErrInvalidReference(repo.Reference)
 	}
 
+	// add both pull, push and delete scope hints for dst repository to save potential delete-scope token requests during deleting
+	hints := []string{auth.ActionPull, auth.ActionPush, auth.ActionDelete}
```

Comment:
According to the code context, it seems `auth.ActionPush` should be remove in both the code and the comment.

Reply:
Looks like we can perform similar optimization for referrers tag schema process on `oras-go` side as well. See https://github.com/oras-project/oras-go/blob/main/registry/remote/repository.go#L1116-L1125
/cc @shizhMSFT

Reply:
@Wwwsylvia  Sounds a good idea. Could you create an issue in `oras-go`?

Reply:
Issue created: https://github.com/oras-project/oras-go/issues/594